src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/transformers/CoroutineSupportTransformers.kt

```diff
@@ -22,22 +23,30 @@ import sun.nio.ch.lincheck.*
 internal class CoroutineCancellabilitySupportTransformer(
     mv: MethodVisitor,
     access: Int,
+    val className: String?,
     methodName: String?,
-    descriptor: String?
-) : AdviceAdapter(ASM_API, mv, access, methodName, descriptor) {
+    desc: String?
+) : AdviceAdapter(ASM_API, mv, access, methodName, desc) {
 
-    override fun visitMethodInsn(opcode: Int, owner: String, name: String, desc: String, itf: Boolean) {
-        if (isCancellableContinuationGetResultMethod(owner, name)) {
+    override fun visitMethodInsn(
+        opcodeAndSource: Int,
+        className: String?,
+        methodName: String?,
+        descriptor: String?,
```

Comment:
I've noticed that here "desc" was renamed to "descriptor", while on line 28 "descriptor" was renamed to "desc"

Reply:
Fair. I renamed all the arguments of `CoroutineCancellabilitySupportTransformer::visitMethodInsn` in accordance with the naming scheme used in all the other transformers:

```
override fun visitMethodInsn(opcode: Int, owner: String, name: String, desc: String, itf: Boolean)
```